source/includes/_edge_webhook.md

```diff
@@ -27,7 +27,11 @@ Webhook Components:
 - Which lead types should trigger a webhook?
   - Leads can be filtered to only send Calls, only Chats, only Forms & Emails, or any combination of these three options.
 - Payload Format
-  - The format that we will send the data in. This can either be sent in JSON, which is the standard format, or in ADF/XML.
-- The webhook can be set to Active by checking the box next to "Active". If left unchecked, the webhook will not send data.
+  - The [structure of the data in the webhook](#lead-info-apis-webhooks-contacts-lead-apis-response-data-details-nbsp) is the same as that returned by the API. This can either be sent in JSON, which is the standard format, or in ADF/XML.
```

Comment:
Where does this go? `[structure of the data in the webhook](#lead-info-apis-webhooks-contacts-lead-apis-response-data-details-nbsp)`

Reply:
https://doc.api.localiq.com/#lead-info-apis-webhooks-contacts-lead-apis-contact-interactions-leads

Reply:
I think you want to reconsider if you look at the length of the section I linked to